docs/dictionary-cache.md

```diff
@@ -0,0 +1,47 @@
+# Dictionary cache 
+
+After Percona XtraBackup takes the [backup](./how-xtrabackup-works.md#back-up) phase, in the [prepare](./how-xtrabackup-works.md#prepare) phase, `InnoDB` performs the crash recovery operation, as if the server exited.
+
+The `--prepare` phase has the following operations:
+
+* Applies the redo log
+* Applies the undo log
+
+Percona XtraBackup physically modifies a specific offset of a page within a tablespace (IBD file) when applying a redo log. In the redo log operation, there is no concept of row or transaction. 
+
+As a logical operation, Percona XtraBackup applies the undo log on a specific row. When the redo log completes, XtraBackup uses the undo log to roll back changes from uncommitted transactions during the backup.
+
+## Undo log
+
+There are two types of undo log records:
+
+* INSERT
+* UPDATE
+
+An undo log record contains a `table_id`. Percona XtraBackup uses this `table_id` to find the table definition, and that information is used to parse the records on an index page. The transaction rollback reads the undo log records and applies the changes. 
```

Comment:
and that information is used to parse the records on an index page. ->  and then uses that information to parse the records on an index page.